vlttng/vlttng_cli.py

```diff
@@ -79,6 +79,13 @@ def _parse_args():
     return args
 
 
+def _jobs_default():
```

Comment:
Just set a `default_jobs` variable once at the beginning of `_parse_args()` instead of calling it twice.